### PR TITLE
Pphw dox review

### PR DESCRIPTION
**Pull request by gonuke**
A final review of these docs

### CODE REVIEW

File: DEPENDENCIES.rst [hunk 1]
```diff
@@ -69,10 +69,10 @@ manager.
 Installing Dependencies (Linux and Unix)
 ----------------------------------------
 
-This guide assumes that the user has root access (to issue sudo commands) and
+This guide assumes that the user has root access (to issue `sudo` commands) and
 access to a package manager or has some other suitable method of automatically
 installing established libraries. This process was tested using a fresh install
-of Ubuntu 12.10 using apt-get as the package manager (scroll down further for
+of Ubuntu versions 12.10, 14.04 and 16.04, using `apt-get` as the package manager (scroll down further for
```
Comment (gonuke) [inline DEPENDENCIES.rst]: I added some versions of Ubuntu here - we may want to remove the 12.10 reference

File: DEPENDENCIES.rst [hunk 2]
```diff
@@ -141,15 +141,15 @@ Installing Dependencies (Mac OSX)
...
 
-This guide assumes that the user has root access (to issue sudo commands) and
+This guide assumes that the user has root access (to issue `sudo` commands) and
 access to a package manager or has some other suitable method of automatically
 installing established libraries. This process was tested using a fresh install
-of Yosemite 10.10.2 using macports as the package manager.  Macports installs
-packages in /opt/local.  If installing to a different location,
-(i.e. /usr/local) change paths in the following instructions accordingly. If
-you use homebrew, try the following instructions with brew commands in place of
+of Yosemite 10.10.2 using `macports` as the package manager.  Macports installs
```
Comment (gonuke) [inline DEPENDENCIES.rst]: Are there other versions of Mac OSX we should be referring to?

File: INSTALL.rst [hunk 3]
```diff
@@ -50,13 +47,14 @@ Run the install script:
   
   python install.py
 
-If you successfully followed the instruction above cyclus binary have been
-generated and be placed in the  ``.local/`` in your home directory. 
-You need to had ``~/.local/bin`` to the bottom of your ``$PATH``:
+If you successfully followed the instruction above, the cyclus binary has been
+generated and placed in the  ``~/.local/`` in your home directory. 
+You need to add ``~/.local/bin`` to the front of your ``$PATH``:
```
Comment (gonuke) [inline INSTALL.rst]: This initially referred to the "bottom" of the path, but the example appears to be the front (top?)

File: DEPENDENCIES.rst (hunk 1 above)
Comment (Baaaaam) [inline DEPENDENCIES.rst]: yes I think we should.

File: DEPENDENCIES.rst (hunk 2 above)
Comment (Baaaaam) [inline DEPENDENCIES.rst]: it should be working at least on:
- OS X 10.9 Mavericks (not tested)
- OS X 10.10 Yosemite (tested by @mbmcgarry) 
- OS X 10.11 El Captain (tested by @baaaaam)
- OS X 10.12 Sierra (not tested)
previous version might be working too

**Pull request merged**